app/client/views/create-set.html

```diff
@@ -7,41 +7,57 @@
 
 		<p class="lead">
 		Set Name
-		<input ng-model = "name" id="Name" type="text" class="form-control input-lg" placeholder="Name"><br>
+		<input ng-model = "set.Name" id="Name" type="text" class="form-control input-lg" placeholder="Name"><br>
 
 		Category
-		<input ng-model = "category" id="Category" type="text" class="form-control input-lg" placeholder="Category"><br>
-		</p>
+		<input ng-model = "set.Category" id="Category" type="text" class="form-control input-lg" placeholder="Category"><br>
 
-		<!--
+		
 		Author
-		<input ng-model = "author" id="Author" type="text" class="form-control input-lg" placeholder="Author"><br>
-		-->
+		<input ng-model = "set.Author" id="Author" type="text" class="form-control input-lg" placeholder="Author"><br>
+		</p>
 
 		<div id="dynamicInput">
 			<p class="lead">
 			Card
 			<br>
-			<input ng-model = "front1" class="form-control input-lg" type="text" name="myInputs[]" placeholder="Front">
-			<input ng-model = "back1" class="form-control input-lg" type="text" name="myInputs[]" placeholder="Back">
+			<input ng-model = "card1.front" class="form-control input-lg" type="text" name="myInputs[]" placeholder="Front">
+			<input ng-model = "card1.back" class="form-control input-lg" type="text" name="myInputs[]" placeholder="Back">
```

Comment:
@anthonyabsher so all you gotta do is push each of the "card" objects (card1, card2, card3, etc.) into the "cards" array in "sets"